doc/user_guide.md

```diff
@@ -0,0 +1,117 @@
+# Exasol SQL Statement Builder User Guide
+
+You can follow this guide to learn more about how to use the SQL Statement
+Builder. It provides explanation of currently supported features with various
+code examples.
+
+## Overview
+
+The Exasol SQL Statement Builder abstracts programmatic creation of SQL
+statements and is intended to replace ubiquitous string concatenation solutions
+which make the code hard to read and are prone to errors and security risks.
+
+SQL Statement Builder allows users to create a SQL statement classes using
+[fluent programming][fluent] that can render into a string.
```

Comment:
" ... programming.

Once you have such an abstract statement, you can for example use the SSB to render the statement into an SQL string."

Reply:
fixed